kiota_abstractions/base_request_configuration.py

```diff
@@ -23,3 +24,11 @@ class BaseRequestConfiguration:
     options: Optional[List[RequestOption]] = None
 
     query_parameters: Optional[QueryParameters] = None
+
+
+@dataclass
+class BaseRequestConfiguration:
```

Comment:
I believe it should inherit from RequestConfiguration to maintain compatibility with older generated clients